pkg/security/probe/probe.go

```diff
@@ -197,15 +249,20 @@ func (p *Probe) AddCustomEventHandler(eventType model.EventType, handler CustomE
 	return nil
 }
 
-func (p *Probe) sendEventToWildcardHandlers(event *model.Event) {
-	for _, handler := range p.fullAccessEventHandlers[model.UnknownEventType] {
+func (p *Probe) sendEventToHandlers(event *model.Event) {
+	for _, handler := range p.eventHandlers[model.UnknownEventType] {
 		handler.HandleEvent(event)
 	}
 }
 
-func (p *Probe) sendEventToSpecificEventTypeHandlers(event *model.Event) {
-	for _, handler := range p.eventHandlers[event.GetEventType()] {
-		handler.HandleEvent(handler.Copy(event))
+func (p *Probe) sendEventToConsumers(event *model.Event) {
+	for _, pc := range p.eventConsumers[event.GetEventType()] {
+		if copied := pc.consumer.Copy(event); copied != nil {
+			select {
+			case pc.eventCh <- copied:
+			default:
```

Comment:
Some metrics for dropped events would be nice. Both here and if `copied == nil`.